docs/en/integrations/language-clients/java/index.md

```diff
@@ -25,7 +25,7 @@ Provides the most flexible and performant way to integrate your app with ClickHo
 
 | Client version | ClickHouse  |
 |----------------|-------------|
-| 0.4.6          | 20.7+       |
+| 0.5.0          | 20.7+       |
```

Comment:
Did we test with such an old version of ClickHouse? v23.3 is the last supported version of ClickHouse https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md

Reply:
No missed it. I will change it

Reply:
I will update with 22.8 since we run tests with that version